aspnetcore/performance/caching/memory.md

```diff
@@ -86,6 +96,32 @@ The following sample:
 
 [!code-csharp[](memory/sample/WebCache/Controllers/HomeController.cs?name=snippet_et&highlight=14-21)]
 
+::: moniker range=">= aspnetcore-2.0"
+
+## Use SetSize, Size, and SizeLimit to limit cache size
+
+The following code creates a unitless fixed size [MemoryCache](/dotnet/api/microsoft.extensions.caching.memory.memorycache?view=aspnetcore-2.1):
```

Comment:
The following code creates a unitless fixed size suitable for use in DI. (Otherwise you wouldn't need the wrapper class).

Reply:
A MemoryCache instance may optionally specify and enforce a size limit. This limit does not have a defined unit of measure because the cache has no mechanic to measure the size of entries. If the limit is set then all entries must specify their own size using their own measuring units.

One example would be if you were primarily caching strings and setting the size of each entry to the length of the string. Another would be to specify the size of all entries as 1, and thus the size limit becomes a limit on the count of entries.